modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -38,17 +57,57 @@ $ sudo podman cp $(sudo podman create registry.redhat.io/rhcam-1-2/openshift-mig
 $ sudo podman cp $(sudo podman create registry.redhat.io/rhcam-1-2/openshift-migration-rhel7-operator:v1.2):/controller-3.yml ./
 ----
 
+ifdef::disconnected-3-4[]
+. Obtain the Operator image value from the `mapping.txt` file that was created when you ran the `oc adm catalog mirror` on the {product-title} 4 cluster:
++
+----
+$ grep openshift-migration-rhel7-operator ./mapping.txt | grep rhcam-1-2
+----
++
+The output shows the mapping between the `registry.redhat.io` image and your mirror registry image:
++
+----
+registry.redhat.io/rhcam-1-2/openshift-migration-rhel7-operator@sha256:468a6126f73b1ee12085ca53a312d1f96ef5a2ca03442bcb63724af5e2614e8a=<registry.apps.example.com>/rhcam-1-2/openshift-migration-rhel7-operator
```

Comment:
Just to double-check, do we want rhel7 images here or rhel8?

Reply:
RHEL 7. That's the image. Nothing to do with what current rhel version is.